mne/viz/backends/renderer.py

```diff
@@ -169,9 +169,13 @@ def _get_3d_backend():
                     MNE_3D_BACKEND = name
                     break
             else:
+                pip_name_map = {'pyvistaqt': 'pyvistaqt',
+                                'notebook': 'ipywidgets'}
                 raise RuntimeError(
-                    'Could not load any valid 3D backend:\n' +
-                    "\n".join(f'{key}: {val}' for key, val in errors.items()))
+                    'Could not load any valid 3D backend,'
+                    ' please install one of the following backends:\n' +
+                    "\n".join(f'{key}: {val} - pip install {pip_name_map[key]}'
```

Comment:
same comment as above: need both `pip` and `conda`

Reply:
I added conda instructions here as well and each instruction is on a new line. The code already in place was a bit complex for me to work around, so apologies if my approach was not so elegant.